e2e/specs/typography.spec.js

```diff
@@ -26,13 +26,45 @@ describe("the typography in the main block", () => {
       .invoke("css", "opacity", "1");
   });
 
-  it("matches the snapshot", () => {
-    cy.get(".main").matchThemedSnapshots("main-block");
+  it("matches the snapshot for main vs sidebar", () => {
+    cy.get("body").matchThemedSnapshots("main-vs-sidebar");
+  });
+
+  it("matches the snapshot for single st.markdown", () => {
+    getElementNextToText("Headers in single st.markdown").matchThemedSnapshots(
+      "single-markdown"
+    );
+  });
+
+  it("matches the snapshot for multiple st.markdown", () => {
+    getElementNextToText(
+      "Headers in multiple st.markdown"
+    ).matchThemedSnapshots("multiple-markdown");
```

Comment:
I think this isn't doing quite what's intended as it only gets the immediate next header and not all of them

Reply:
Fixed. (This was fixed yesterday, but I forgot to update the PR description to reflect that)